cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -295,28 +295,59 @@ async function fetchAnnotations(frameNumber?: number): Promise<{
     };
 }
 
+function navigationInterruptionDetected(callback: (interrupted: boolean) => void): void {
+    let currentPath = window.location.pathname;
+
+    // Listen for popstate events (back/forward navigation)
+    window.addEventListener('popstate', () => {
```

Comment:
``popstate`` event listener is never removed. Memory leak, because `fetchAnnotationsAsync` may be called plenty of times.

Reply:
Ok, I should remove the event listener after it's triggered